docs/static/spec/openapi/logstash-api.yaml

```
@@ -141,24 +306,12 @@ paths:
                       - name: logstash-codec-dots
                         version: 3.0.2
                       - name: logstash-coded-edn
-                        version: 3.0.2 
+                        version: 3.0.2
 
   /_node/stats:
     get:
       summary: Get node stats
```

The other summaries in this group use "statistics" rather than "stats" so they should ideally match.
```suggestion
      summary: Get node statistics
```